Kill THSize.

gchanan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #31254 Kill THTensor_(bhistc).
* **#31218 Kill THSize.**
* #31217 Kill THLogAdd
* #31216 Kill some unnecessary function declarations.

It isn't used.

Differential Revision: [D18986641](https://our.internmc.facebook.com/intern/diff/D18986641)

gchanan committed:
Kill THSize.

It isn't used.

[ghstack-poisoned]

kostmo commented:
## CircleCI build failures summary
As of commit 903f468:
* **11/11** failures introduced in this PR



## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=903f4687cc07c5438bb67d9bf30cc27a09977c11).

### 10 new failures recognized by patterns
The following build failures don't appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934133) binary_linux_libtorch_2_7m_cpu_devtoolset7_shared-with-deps_build (1/10)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73644949))
```
Dec 12 23:49:15 [ 51%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/MemoryOverlap.cpp.o 
Dec 12 23:49:15 [ 51%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ParallelCommon.cpp.o 
Dec 12 23:49:15 [ 51%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ParallelNative.cpp.o 
Dec 12 23:49:15 [ 51%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ParallelOpenMP.cpp.o 
Dec 12 23:49:15 [ 51%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ParallelThreadPoolNative.cpp.o 
Dec 12 23:49:15 [ 51%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/NamedTensorUtils.cpp.o 
Dec 12 23:49:15 [ 51%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ParallelNativeTBB.cpp.o 
Dec 12 23:49:15 [ 51%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/SparseTensorImpl.cpp.o 
Dec 12 23:49:15 [ 51%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/TensorGeometry.cpp.o 
Dec 12 23:49:16 In file included from /pytorch/aten/src/ATen/Context.cpp:16:0: 
Dec 12 23:49:16 /pytorch/aten/src/TH/TH.h:12:10: fatal error: TH/THLogAdd.h: No such file or directory 
Dec 12 23:49:16  #include <TH/THLogAdd.h> 
Dec 12 23:49:16           ^~~~~~~~~~~~~~~ 
Dec 12 23:49:16 compilation terminated. 
Dec 12 23:49:16 gmake[2]: *** [caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o] Error 1 
Dec 12 23:49:16 gmake[2]: *** Waiting for unfinished jobs.... 
Dec 12 23:49:22 gmake[1]: *** [caffe2/CMakeFiles/torch_cpu.dir/all] Error 2 
Dec 12 23:49:22 gmake: *** [all] Error 2 
Dec 12 23:49:22 Traceback (most recent call last): 
Dec 12 23:49:22   File "setup.py", line 755, in <module> 
Dec 12 23:49:22     build_deps() 
```
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934134) binary_linux_manywheel_2_7mu_cpu_devtoolset7_build (2/10)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73644964))
```
Dec 12 23:49:13 [ 49%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/MemoryOverlap.cpp.o 
Dec 12 23:49:13 [ 49%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/NamedTensorUtils.cpp.o 
Dec 12 23:49:13 [ 49%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ParallelCommon.cpp.o 
Dec 12 23:49:13 [ 49%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ParallelNative.cpp.o 
Dec 12 23:49:13 [ 49%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ParallelNativeTBB.cpp.o 
Dec 12 23:49:13 [ 49%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ParallelOpenMP.cpp.o 
Dec 12 23:49:13 [ 49%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ParallelThreadPoolNative.cpp.o 
Dec 12 23:49:13 [ 49%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/SparseTensorImpl.cpp.o 
Dec 12 23:49:13 [ 49%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/TensorGeometry.cpp.o 
Dec 12 23:49:14 In file included from /pytorch/aten/src/ATen/Context.cpp:16:0: 
Dec 12 23:49:14 /pytorch/aten/src/TH/TH.h:12:10: fatal error: TH/THLogAdd.h: No such file or directory 
Dec 12 23:49:14  #include <TH/THLogAdd.h> 
Dec 12 23:49:14           ^~~~~~~~~~~~~~~ 
Dec 12 23:49:14 compilation terminated. 
Dec 12 23:49:14 gmake[2]: *** [caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o] Error 1 
Dec 12 23:49:14 gmake[2]: *** Waiting for unfinished jobs.... 
Dec 12 23:49:21 gmake[1]: *** [caffe2/CMakeFiles/torch_cpu.dir/all] Error 2 
Dec 12 23:49:21 gmake: *** [all] Error 2 
Dec 12 23:49:21 Traceback (most recent call last): 
Dec 12 23:49:21   File "setup.py", line 755, in <module> 
Dec 12 23:49:21     build_deps() 
```
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934124) caffe2_onnx_py3_6_clang7_ubuntu16_04_build (3/10)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73644965))
```
Dec 12 23:49:11 [ 48%] Building CXX object third_party/onnx/CMakeFiles/onnx.dir/onnx/common/assertions.cc.o 
Dec 12 23:49:11 [ 48%] Building CXX object third_party/onnx/CMakeFiles/onnx.dir/onnx/common/model_helpers.cc.o 
Dec 12 23:49:11 [ 48%] Linking CXX static library ../../lib/libonnx.a 
Dec 12 23:49:12 [ 48%] Built target onnx 
Dec 12 23:49:23 Scanning dependencies of target torch_cpu 
Dec 12 23:49:24 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ThreadLocalDebugInfo.cpp.o 
Dec 12 23:49:24 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/DynamicLibrary.cpp.o 
Dec 12 23:49:24 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o 
Dec 12 23:49:24 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/DLConvertor.cpp.o 
Dec 12 23:49:24 In file included from /var/lib/jenkins/workspace/aten/src/ATen/Context.cpp:16: 
Dec 12 23:49:24 /var/lib/jenkins/workspace/aten/src/TH/TH.h:12:10: fatal error: 'TH/THLogAdd.h' file not found 
Dec 12 23:49:24 #include <TH/THLogAdd.h> 
Dec 12 23:49:24          ^~~~~~~~~~~~~~~ 
Dec 12 23:49:24 1 error generated. 
Dec 12 23:49:24 make[2]: *** [caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o] Error 1 
Dec 12 23:49:24 make[2]: *** Waiting for unfinished jobs.... 
Dec 12 23:49:24 caffe2/CMakeFiles/torch_cpu.dir/build.make:298: recipe for target 'caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o' failed 
Dec 12 23:49:24 make[1]: *** [caffe2/CMakeFiles/torch_cpu.dir/all] Error 2 
Dec 12 23:49:24 CMakeFiles/Makefile2:5548: recipe for target 'caffe2/CMakeFiles/torch_cpu.dir/all' failed 
Dec 12 23:49:24 make: *** [all] Error 2 
Dec 12 23:49:24 Makefile:138: recipe for target 'all' failed 
```
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934126) pytorch_linux_xenial_py2_7_9_build (4/10)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73644966))
```
Dec 12 23:47:17 +++ eval 'extract_trap_cmd ' 
Dec 12 23:47:17 ++++ extract_trap_cmd 
Dec 12 23:47:17 ++++ printf '%s\n' '' 
Dec 12 23:47:17 +++ printf '%s\n' cleanup 
Dec 12 23:47:17 ++ trap -- ' 
Dec 12 23:47:17 cleanup' EXIT 
Dec 12 23:47:17 ++ which sccache 
Dec 12 23:47:17 ++ sccache --stop-server 
Dec 12 23:47:17 Stopping sccache server... 
Dec 12 23:47:17 error: couldn't connect to server 
Dec 12 23:47:17 caused by: Connection refused (os error 111) 
Dec 12 23:47:17 ++ true 
Dec 12 23:47:17 ++ rm /var/lib/jenkins/sccache_error.log 
Dec 12 23:47:17 rm: cannot remove '/var/lib/jenkins/sccache_error.log': No such file or directory 
Dec 12 23:47:17 ++ true 
Dec 12 23:47:17 ++ SCCACHE_ERROR_LOG=/var/lib/jenkins/sccache_error.log 
Dec 12 23:47:17 ++ SCCACHE_IDLE_TIMEOUT=1200 
Dec 12 23:47:17 ++ RUST_LOG=sccache::server=error 
Dec 12 23:47:17 ++ sccache --start-server 
Dec 12 23:47:18 Starting sccache server... 
Dec 12 23:47:18 ++ sccache --zero-stats 
```
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934128) caffe2_onnx_py2_gcc5_ubuntu16_04_build (5/10)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73645286))
```
Dec 12 23:49:34 [ 48%] Building CXX object third_party/onnx/CMakeFiles/onnx.dir/onnx/common/assertions.cc.o 
Dec 12 23:49:34 [ 48%] Building CXX object third_party/onnx/CMakeFiles/onnx.dir/onnx/common/model_helpers.cc.o 
Dec 12 23:49:35 [ 48%] Linking CXX static library ../../lib/libonnx.a 
Dec 12 23:49:35 [ 48%] Built target onnx 
Dec 12 23:49:47 Scanning dependencies of target torch_cpu 
Dec 12 23:49:48 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/DynamicLibrary.cpp.o 
Dec 12 23:49:48 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ThreadLocalDebugInfo.cpp.o 
Dec 12 23:49:48 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o 
Dec 12 23:49:48 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/DLConvertor.cpp.o 
Dec 12 23:49:48 In file included from /var/lib/jenkins/workspace/aten/src/ATen/Context.cpp:16:0: 
Dec 12 23:49:48 /var/lib/jenkins/workspace/aten/src/TH/TH.h:12:25: fatal error: TH/THLogAdd.h: No such file or directory 
Dec 12 23:49:48 compilation terminated. 
Dec 12 23:49:48 caffe2/CMakeFiles/torch_cpu.dir/build.make:298: recipe for target 'caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o' failed 
Dec 12 23:49:48 make[2]: *** [caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o] Error 1 
Dec 12 23:49:48 make[2]: *** Waiting for unfinished jobs.... 
Dec 12 23:49:48 [ 49%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/TensorUtils.cpp.o 
Dec 12 23:49:49 make[1]: *** [caffe2/CMakeFiles/torch_cpu.dir/all] Error 2 
Dec 12 23:49:49 CMakeFiles/Makefile2:5422: recipe for target 'caffe2/CMakeFiles/torch_cpu.dir/all' failed 
Dec 12 23:49:49 make: *** [all] Error 2 
Dec 12 23:49:49 Makefile:138: recipe for target 'all' failed 
Dec 12 23:49:49 Traceback (most recent call last): 
```
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934140) pytorch_linux_xenial_py3_6_gcc5_4_build (6/10)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73645599))
```
Dec 12 23:48:06 +++ eval 'extract_trap_cmd ' 
Dec 12 23:48:06 ++++ extract_trap_cmd 
Dec 12 23:48:06 ++++ printf '%s\n' '' 
Dec 12 23:48:06 +++ printf '%s\n' cleanup 
Dec 12 23:48:06 ++ trap -- ' 
Dec 12 23:48:06 cleanup' EXIT 
Dec 12 23:48:06 ++ which sccache 
Dec 12 23:48:06 ++ sccache --stop-server 
Dec 12 23:48:06 Stopping sccache server... 
Dec 12 23:48:06 error: couldn't connect to server 
Dec 12 23:48:06 caused by: Connection refused (os error 111) 
Dec 12 23:48:06 ++ true 
Dec 12 23:48:06 ++ rm /var/lib/jenkins/sccache_error.log 
Dec 12 23:48:06 rm: cannot remove '/var/lib/jenkins/sccache_error.log': No such file or directory 
Dec 12 23:48:06 ++ true 
Dec 12 23:48:06 ++ SCCACHE_ERROR_LOG=/var/lib/jenkins/sccache_error.log 
Dec 12 23:48:06 ++ SCCACHE_IDLE_TIMEOUT=1200 
Dec 12 23:48:06 ++ RUST_LOG=sccache::server=error 
Dec 12 23:48:06 ++ sccache --start-server 
Dec 12 23:48:06 Starting sccache server... 
Dec 12 23:48:06 ++ sccache --zero-stats 
```
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934116) pytorch_linux_xenial_py3_clang5_mobile_build (7/10)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73645658))
```
Dec 12 23:49:18 +++ eval 'extract_trap_cmd ' 
Dec 12 23:49:18 ++++ extract_trap_cmd 
Dec 12 23:49:18 ++++ printf '%s\n' '' 
Dec 12 23:49:18 +++ printf '%s\n' cleanup 
Dec 12 23:49:18 ++ trap -- ' 
Dec 12 23:49:18 cleanup' EXIT 
Dec 12 23:49:18 ++ which sccache 
Dec 12 23:49:18 ++ sccache --stop-server 
Dec 12 23:49:18 Stopping sccache server... 
Dec 12 23:49:18 error: couldn't connect to server 
Dec 12 23:49:18 caused by: Connection refused (os error 111) 
Dec 12 23:49:18 ++ true 
Dec 12 23:49:18 ++ rm /var/lib/jenkins/sccache_error.log 
Dec 12 23:49:18 rm: cannot remove '/var/lib/jenkins/sccache_error.log': No such file or directory 
Dec 12 23:49:18 ++ true 
Dec 12 23:49:18 ++ SCCACHE_ERROR_LOG=/var/lib/jenkins/sccache_error.log 
Dec 12 23:49:18 ++ SCCACHE_IDLE_TIMEOUT=1200 
Dec 12 23:49:18 ++ RUST_LOG=sccache::server=error 
Dec 12 23:49:18 ++ sccache --start-server 
Dec 12 23:49:18 Starting sccache server... 
Dec 12 23:49:18 ++ sccache --zero-stats 
```
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934130) pytorch_linux_xenial_py3_clang5_asan_build (8/10)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73645808))
```
Dec 12 23:48:16 +++ eval 'extract_trap_cmd ' 
Dec 12 23:48:16 ++++ extract_trap_cmd 
Dec 12 23:48:16 ++++ printf '%s\n' '' 
Dec 12 23:48:16 +++ printf '%s\n' cleanup 
Dec 12 23:48:16 ++ trap -- ' 
Dec 12 23:48:16 cleanup' EXIT 
Dec 12 23:48:16 ++ which sccache 
Dec 12 23:48:16 ++ sccache --stop-server 
Dec 12 23:48:16 Stopping sccache server... 
Dec 12 23:48:16 error: couldn't connect to server 
Dec 12 23:48:16 caused by: Connection refused (os error 111) 
Dec 12 23:48:16 ++ true 
Dec 12 23:48:16 ++ rm /var/lib/jenkins/sccache_error.log 
Dec 12 23:48:16 rm: cannot remove '/var/lib/jenkins/sccache_error.log': No such file or directory 
Dec 12 23:48:16 ++ true 
Dec 12 23:48:16 ++ SCCACHE_ERROR_LOG=/var/lib/jenkins/sccache_error.log 
Dec 12 23:48:16 ++ SCCACHE_IDLE_TIMEOUT=1200 
Dec 12 23:48:16 ++ RUST_LOG=sccache::server=error 
Dec 12 23:48:16 ++ sccache --start-server 
Dec 12 23:48:16 Starting sccache server... 
Dec 12 23:48:16 ++ sccache --zero-stats 
```
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934139) binary_linux_libtorch_2_7m_cpu_gcc5_4_cxx11-abi_shared-with-deps_build (9/10)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73646050))
```
Dec 12 23:49:55 [1572/2991] Building CXX object caffe2/CMakeFiles/caffe2_pybind11_state.dir/python/pybind_state_nomni.cc.o 
Dec 12 23:49:56 [1573/2991] Building CXX object caffe2/CMakeFiles/utility_ops_test.dir/operators/utility_ops_test.cc.o 
Dec 12 23:49:56 [1574/2991] Building CXX object caffe2/CMakeFiles/NamedTensor_test.dir/__/aten/src/ATen/test/NamedTensor_test.cpp.o 
Dec 12 23:49:58 [1575/2991] Building CXX object caffe2/CMakeFiles/thread_init_test.dir/__/aten/src/ATen/test/thread_init_test.cpp.o 
Dec 12 23:49:59 [1576/2991] Building CXX object caffe2/CMakeFiles/smart_tensor_printer_test.dir/utils/smart_tensor_printer_test.cc.o 
Dec 12 23:50:00 [1577/2991] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/CPUGenerator.cpp.o 
Dec 12 23:50:01 [1578/2991] Building CXX object caffe2/CMakeFiles/tensor_interop_test.dir/__/aten/src/ATen/test/tensor_interop_test.cpp.o 
Dec 12 23:50:01 [1579/2991] Building CXX object caffe2/CMakeFiles/reduce_ops_test.dir/__/aten/src/ATen/test/reduce_ops_test.cpp.o 
Dec 12 23:50:01 [1580/2991] Building CXX object caffe2/CMakeFiles/scalar_tensor_test.dir/__/aten/src/ATen/test/scalar_tensor_test.cpp.o 
Dec 12 23:50:02 [1581/2991] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o 
Dec 12 23:50:02 FAILED: caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o  
AVX -DUSE_AVX2 -DTH_HAVE_THREAD -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wno-write-strings -Wno-unknown-pragmas -Wno-missing-braces -Wno-maybe-uninitialized -fvisibility=hidden -O2 -DCAFFE2_BUILD_MAIN_LIB -DASMJIT_STATIC -std=gnu++14 -MD -MT caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o -MF caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o.d -o caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o -c ../aten/src/ATen/Context.cpp 
Dec 12 23:50:02 In file included from ../aten/src/ATen/Context.cpp:16:0: 
Dec 12 23:50:02 ../aten/src/TH/TH.h:12:25: fatal error: TH/THLogAdd.h: No such file or directory 
Dec 12 23:50:02 compilation terminated. 
Dec 12 23:50:02 [1582/2991] Building CXX object caffe2/CMakeFiles/kernel_function_test.dir/__/aten/src/ATen/core/boxing/kernel_function_test.cpp.o 
Dec 12 23:50:02 [1583/2991] Building CXX object caffe2/CMakeFiles/scalar_test.dir/__/aten/src/ATen/test/scalar_test.cpp.o 
Dec 12 23:50:04 [1584/2991] Building CXX object caffe2/CMakeFiles/int8_test.dir/operators/quantized/int8_test.cc.o 
Dec 12 23:50:04 [1585/2991] Building CXX object caffe2/CMakeFiles/tensor_iterator_test.dir/__/aten/src/ATen/test/tensor_iterator_test.cpp.o 
Dec 12 23:50:04 [1586/2991] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/DynamicLibrary.cpp.o 
Dec 12 23:50:06 [1587/2991] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ParallelCommon.cpp.o 
```
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934141) pytorch_xla_linux_xenial_py3_6_clang7_build (10/10)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73646394))
```
Dec 12 23:48:10 +++ eval 'extract_trap_cmd ' 
Dec 12 23:48:10 ++++ extract_trap_cmd 
Dec 12 23:48:10 ++++ printf '%s\n' '' 
Dec 12 23:48:10 +++ printf '%s\n' cleanup 
Dec 12 23:48:10 ++ trap -- ' 
Dec 12 23:48:10 cleanup' EXIT 
Dec 12 23:48:10 ++ which sccache 
Dec 12 23:48:10 ++ sccache --stop-server 
Dec 12 23:48:10 Stopping sccache server... 
Dec 12 23:48:10 error: couldn't connect to server 
Dec 12 23:48:10 caused by: Connection refused (os error 111) 
Dec 12 23:48:10 ++ true 
Dec 12 23:48:10 ++ rm /var/lib/jenkins/sccache_error.log 
Dec 12 23:48:10 rm: cannot remove '/var/lib/jenkins/sccache_error.log': No such file or directory 
Dec 12 23:48:10 ++ true 
Dec 12 23:48:10 ++ SCCACHE_ERROR_LOG=/var/lib/jenkins/sccache_error.log 
Dec 12 23:48:10 ++ SCCACHE_IDLE_TIMEOUT=1200 
Dec 12 23:48:10 ++ RUST_LOG=sccache::server=error 
Dec 12 23:48:10 ++ sccache --start-server 
Dec 12 23:48:10 Starting sccache server... 
Dec 12 23:48:10 ++ sccache --zero-stats 
```


---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

gchanan committed:
Update on "Kill THSize."


It isn't used.

Differential Revision: [D18986641](https://our.internmc.facebook.com/intern/diff/D18986641)

[ghstack-poisoned]
